samples/posix/eventfd/README.rst

```diff
@@ -0,0 +1,46 @@
+.. _posix-eventfd-sample:
+
+POSIX eventfd()
+########################################################
+
+Overview
+********
+
+This sample application demonstrates using the POSIX eventfd()
```

Comment:
adding a reference to eventfd() doc would be welcome here, but not sure why there's none on opengroup webpage? @cfriedt might know

Reply:
`eventfd()` is not yet part of POSIX. It just happens to be lumped in with POSIX in Zephyr because it's available on most POSIX systems, is mostly dependent already on existing POSIX APIs, etc.